src/pages/Integrations/Create/eventTypesStep.ts

```diff
@@ -14,8 +14,7 @@ export const eventTypesStep = () => ({
     {
       component: componentTypes.PLAIN_TEXT,
       name: 'plain-text',
-      label:
-        'Select event types you would like your integration to react to. You may also skip this step and attach event types later.',
+      label: `Select event types you would like your integration to react to. This will create a behavior group automatically for you. You may also skip this step and attach event types later.`,
```

Comment:
```suggestion
      label: 'Select event types you would like your integration to react to. This will create a behavior group automatically for you. You may also skip this step and attach event types later.',
```